javascript-functions-practice/get-last-name-of-person.js

```diff
@@ -0,0 +1,5 @@
+/* exported getLastNameOfPerson */
+function getLastNameOfPerson(person) {
+  var LastNameOfPerson = person.lastName;
```

Comment:
make sure to switch this variable to a lowercase `L`. uppercase characters are only used for special variables.